src/main/java/aspects/UserRestoreAspect.java

```diff
@@ -98,9 +99,7 @@ public void closeRestoreFactory(JoinPoint joinPoint) throws Throwable {
 
     private HqAuth getAuthHeaders(String domain, String username, String sessionToken) {
         HqAuth auth;
-        if (sessionToken != null && sessionToken.equals(authKey)) {
-            auth = new BasicAuth(touchformsUsername, touchformsPassword);
```

Comment:
is this removing the touchforms user auth for formplayer requests to hq? we need to keep that around until all formplayer requests are using the mac digest auth

Reply:
actually, aren't these changes already present in https://github.com/dimagi/formplayer/pull/661?